File: app/src/main/java/com/smedialink/abakarmagomedov/githubapplication/presentation/UserPostActivity/UserPostPresenter.java
package com.smedialink.abakarmagomedov.githubapplication.presentation.UserPostActivity;

import com.smedialink.abakarmagomedov.githubapplication.presentation.UsersActivity.UsersView;

/**
 * Created by abakarmagomedov on 15/05/17.
 */

public interface UserPostPresenter {

    void getRepos(String login);
    void attachView(UserPostView view);
    void detachView();
}

File: app/src/main/java/com/smedialink/abakarmagomedov/githubapplication/presentation/UserPostActivity/UserPostPresenter.kt
package com.smedialink.abakarmagomedov.githubapplication.presentation.UserPostActivity

/**
 * Created by abakarmagomedov on 15/05/17.
 */
interface UserPostPresenter {
    fun getRepos(login: String)
    fun attachView(view: UserPostView)
    fun detachView()
}
